webdriver/tests/element_send_keys/interactability.py

```diff
@@ -45,7 +45,7 @@ def test_document_element_is_interactable(session):
 
     response = element_send_keys(session, element, "foo")
     assert_success(response)
-    assert session.active_element == element
+    assert session.active_element == element or session.active_element == body
```

Comment:
If we go with this approach, depending on @andreastt feedback, you can simplify that to `assert session.active_element in [body, element]`.

Reply:
Change made! Thanks